tests/testthat/test-5prevalence.R

```diff
@@ -385,31 +325,17 @@ test_that("agglomerateByPrevalence", {
                                       rank = "Phylum",
                                       detection = 1/100,
                                       prevalence = 50/100,
-                                      as.relative = TRUE,
                                       other.label = "test")
     expect_s4_class(actual,class(GlobalPatterns))
-    expect_equal(dim(actual),c(6,26))
-    expect_equal(rowData(actual)[6,"Phylum"],"test")
+    expect_equal(dim(actual),c(35,26))
+    expect_equal(rowData(actual)[35,"Phylum"],"test")
 
     actual <- agglomerateByPrevalence(GlobalPatterns,
                                       rank = NULL,
                                       detection = 0.0001,
                                       prevalence = 50/100,
-                                      as.relative = TRUE,
                                       other.label = "test")
-    expect_equal(agglomerateByPrevalence(GlobalPatterns,
-                                           rank = NULL,
-                                           detection = 0.0001,
-                                           prevalence = 50/100,
-                                           as.relative = TRUE,
-                                           other.label = "test"),
-                 agglomerateByPrevalence(GlobalPatterns,
-                                           rank = NULL,
-                                           detection = 0.0001,
-                                           prevalence = 50/100,
-                                           as.relative = TRUE,
-                                           other.label = "test"))
```

Comment:
Does this work with

agglomerateByPrevalence(tse, assay.type = "relabundance", detection = 0.001, prevalence = 0.5)

Are the prevalences calculated based on relative abundance assay? If not, then we have a problem.

Reply:
Hmm, no, unless after `tse <- transformAssay(tse, assay.type = "counts", method = "relabundance")`

Reply:
Yes, that is something that must be done first

Reply:
`agglomerateByPrevalence(tse, assay.type = "relabundance", detection = 0.001, prevalence = 0.5)` works fine and `assay.type `is parsed in `getPrevalence`. is this what I needed to investigate or perhaps some comparison is needed?

Reply:
That is good, thanks